profile/profiler.go

```diff
@@ -437,7 +444,7 @@ func (c *CPUInfo) GetSibling(processor int) (int, error) {
 // SocketCPUs returns a list of processors of the socket
 func (c *CPUInfo) SocketCPUs(socket int) ([]int, error) {
 	var result []int
-	if socket >= len(c.sockets) {
+	if socket >= len(c.sockets) || socket < 0 {
```

Comment:
a negative value doesn't make sense; add a comment

Reply:
Done

Reply:
thanks, please push it for a review

Reply:
Done